extensions/extensions-helm/aissemble-airflow-chart/pom.xml

```diff
@@ -15,31 +15,9 @@
 
     <build>
         <plugins>
-            <!-- The airflow subchart defaults to setting the image tag to the airflow version -->
-            <!-- We use a template here to ease release maintenance.  -->
             <plugin>
                 <groupId>com.google.code.maven-replacer-plugin</groupId>
                 <artifactId>replacer</artifactId>
-                <version>1.5.3</version>
```

Comment:
I: Moved configuration up to the `extensions-helm` POM so we just have to include the plugin to do the version replacement.